communication/rocketchat.go

```diff
@@ -0,0 +1,64 @@
+package communication
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/scribble-rs/scribble.rs/game"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"time"
+)
+
+type rocketChatPayload struct {
+	Alias string `json:"alias"`
+	Text  string `json:"text"`
+}
+
+func updateRocketChat(lobby *game.Lobby, player *game.Player) {
+	state := "connected"
+	players := len(lobby.Players)
```

Comment:
You've made an incorrect assumption here. If a player leaves, his player entry won't be deleted. This is in order to avoid losing score due to lags or short afk times. You have to check how many players have set `Connected` to `true`.